core_registry_node/variables.tf

```diff
@@ -49,7 +49,7 @@ variable "node_instance_type" {
 }
 
 variable "volume_size" {
-  type        = string
+  type        = number
   description = "Root volume size for EC2"
   default     = "300"
```

Comment:
Don't remember if it matters but I'd assume these would be unquoted to treat it as a number

Reply:
Yeah I went back and forth on what to do with that but read https://developer.hashicorp.com/terraform/language/expressions/types#type-conversion and left it quoted.  I think it is fine either way:

>  Terraform automatically converts number and bool values to strings when needed. It also converts strings to numbers or bools, as long as the string contains a valid representation of a number or bool value.
> 
> [true](https://developer.hashicorp.com/terraform/language/expressions/types#true) converts to "true", and vice-versa
> [false](https://developer.hashicorp.com/terraform/language/expressions/types#false) converts to "false", and vice-versa
> [15](https://developer.hashicorp.com/terraform/language/expressions/types#15) converts to "15", and vice-versa

Reply:
I'm changing it to unquoted - want to see if it works and I think that is more technically correct